powerspikegg/frontend/src/app/summoner/matches/matches.component.html

```diff
@@ -0,0 +1,115 @@
+<div class="main-content">
+  <div class="container" fxLayout.lg="column">
+
+    <div class="match defeat" fxLayout.lg="row" fxFlexOffset.lg="2" fxFlex.lg="96" >
+      <div class="champion">
+        <img src="//ddragon.leagueoflegends.com/cdn/6.24.1/img/champion/Veigar.png" alt="Veigar">
```

Comment:
Note for later: that would be nice if we could store all those images on our own servers. Do not fix for the mock.